examples/http-server/main.go

```diff
@@ -27,7 +31,12 @@ func main() {
 	a.GET("/mysql", MysqlHandler)
 
 	// Run the application
-	a.Run()
+	go a.Run()
+
+	ctx, stop := signal.NotifyContext(context.Background(), os.Interrupt, syscall.SIGINT, syscall.SIGTERM)
```

Comment:
I would have expect that this part would be in a subroutine and before the app is launched.

So the app would be without subroutine